components/Locale/interfaces.rst

```diff
@@ -0,0 +1,58 @@
+Interfaces
+==========
+
+Model Interfaces
+----------------
+
+.. _component_locale_model_locale-interface:
+
+LocaleInterface
+~~~~~~~~~~~~~~~
+
+This interface should be implemented by model representing a single Locale.
+
+.. note::
+	This interface extends :ref:`component_resource_model_timestampable-interface` and :ref:`component_resource_model_toggleable-interface` |br|
+    For more detailed information go to `Sylius API LocaleInterface`_.
+
+.. _Sylius API LocaleInterface: http://api.sylius.org/Sylius/Component/Locale/Model/LocaleInterface.html
+
+.. _component_inventory_model_stockable-interface:
+
+LocalesAwareInterface
+~~~~~~~~~~~~~~~~~~~~~
+
+This interface provides basic operations for locale management.
+If you want to have locales in your model just implement this interface.
+
+.. note::
+	For more detailed information go to `Sylius API LocalesAwareInterface`_.
+
+.. _Sylius API LocalesAwareInterface: http://api.sylius.org/Sylius/Component/Locale/Model/LocalesAwareInterface.html
+
+Service Interfaces
+------------------
+
+.. _component_inventory_checker_availability-checker-interface:
+
+LocaleContextInterface
+~~~~~~~~~~~~~~~~~~~~~~
+
+To characterize an object which is a **LocaleContext** it needs to implement the ``LocaleContextInterface``
+
+.. note::
+	For more detailed information go to `Sylius API LocaleContextInterface`_.
+
+.. _Sylius API LocaleContextInterface: http://api.sylius.org/Sylius/Component/Locale/Context/LocaleContextInterface.html
+
+.. _component_inventory_factory_inventory-unit-factory-interface:
+
+LocaleProviderInterface
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To characterize an object which is a **LocaleProvider** it needs to implement the ``LocaleProviderInterface``
```

Comment:
Something like ``This interface is implemented by the service responsible for providing you with a list of available locales`` is much better. What you said here is pretty obvious.